### DISCUSSION TITLE
create new compund from environmental toxicant

### DISCUSSION THREAD

**Discussion by hasnawatiamqam** [Open-Systems-Pharmacology/Forum#1842] [Questions & Problems]
I want to create methylmercury compound. However some physiochemical parameters can not be input because there is no space to input. how to solve the problem?

**Comment by Xiaoyang-svg** [comment]
Hello, have you solved this problem? I have a similar problem and don't know how to deal with it.

**Comment by Proen1** [comment]
For adding a non traditional chemical in the PK-Sim chemical module you need to set the ADME processes yourself. PK-Sim using internal QSAR based on lipophilicity, pKa and MW but they are just applicable to organic molecules. In you case you need to fill the chemical properties to the best of your knowledge , for parameters like logP, you can just give a prop value ( you  will then overwrite the ADME parameters based on lipophlicity). Once you set the simulation , change directly the tissues partition coefficients ,  tissues permeability and oral absorption to the values of your chemical . I am not sure what other parameters you have for methyl mercury?
